File: Projects/ThematicCalculator/app/src/main/java/dam_a45125/thematiccalculator/MainActivity.java
package dam_a45125.thematiccalculator;

import androidx.appcompat.app.AppCompatActivity;
import androidx.core.content.ContextCompat;
import android.content.Intent;
import android.graphics.drawable.AnimationDrawable;
import android.os.Bundle;
import android.view.MotionEvent;
import android.view.View;
import android.widget.ImageButton;
import android.widget.Button;
import android.widget.ImageView;
import android.widget.TextView;

public class MainActivity extends AppCompatActivity implements View.OnClickListener{

    private ImageButton slbIcon, scpIcon, portoIcon;
    private Button bSoma, bSub, bMult, bDiv, bCosseno, bSeno, bMod, bTangente,setButton,getButton;
    private Calculator calc;
    private AnimationDrawable animDraw,animDraw2;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        slbIcon = (ImageButton) findViewById(R.id.imageButtonSLB);
        scpIcon = (ImageButton) findViewById(R.id.imageButtonSCP);
        portoIcon = (ImageButton) findViewById(R.id.imageButtonPorto);

        bSoma = (Button) this.findViewById(R.id.buttonSoma);
        bSub = (Button) this.findViewById(R.id.buttonSubtracao);
        bMult = (Button) this.findViewById(R.id.buttonMult);
        bDiv = (Button) this.findViewById(R.id.buttonDiv);
        bCosseno = (Button) this.findViewById(R.id.buttonCos);
        bSeno = (Button) this.findViewById(R.id.buttonSeno);
        bTangente = (Button) this.findViewById(R.id.buttonTangente);
        bMod = (Button) this.findViewById(R.id.buttonModule);
        setButton = (Button) this.findViewById(R.id.buttonGetOperador1);
        getButton = (Button) this.findViewById(R.id.buttonSetOperador1);
        calc = new Calculator();

        ImageView imageView = (ImageView)findViewById(R.id.imageAnim);
        imageView.setBackgroundResource(R.drawable.animation);
        animDraw = (AnimationDrawable)imageView.getBackground();

        ImageView imageView2 = (ImageView)findViewById(R.id.imageAnim2);
        imageView2.setBackgroundResource(R.drawable.animations_network);
        animDraw2 = (AnimationDrawable)imageView2.getBackground();

        slbIcon.setOnTouchListener(new View.OnTouchListener() {
            @Override
            public boolean onTouch(View arg0, MotionEvent arg1) {
                setContext(R.id.imageButtonSLB);
                return false;
            }
        });

        scpIcon.setOnTouchListener(new View.OnTouchListener() {
            @Override
            public boolean onTouch(View v, MotionEvent event) {
                setContext(R.id.imageButtonSCP);
                return false;
            }
        });

        portoIcon.setOnTouchListener(new View.OnTouchListener() {
            @Override
            public boolean onTouch(View v, MotionEvent event) {
                setContext(R.id.imageButtonPorto);
                return false;
            }
        });
        bSoma.setOnClickListener(this);
        bSub.setOnClickListener(this);
        bMult.setOnClickListener(this);
        bDiv.setOnClickListener(this);
        bCosseno.setOnClickListener(this);
        bSeno.setOnClickListener(this);
        bTangente.setOnClickListener(this);
        bMod.setOnClickListener(this);
        setButton.setOnClickListener(this);
        getButton.setOnClickListener(this);
    }

    public void onClick(View v) {
        Calculator calc = new Calculator();
        TextView number1EditText = findViewById(R.id.editText);
        TextView number2EditText = findViewById(R.id.editText2);
        TextView number3EditText = findViewById(R.id.textResult);

        String sNum1 = number1EditText.getText().toString();
        String sNum2 = number2EditText.getText().toString();
        double num1 = getDouble(sNum1);
        double num2 = getDouble(sNum2);
        Button b = (Button) v;
        double value = 0;
        if (b.getId() == R.id.buttonSoma) {
            value = calc.sum(num1, num2);
        } else if (b.getId() == R.id.buttonSubtracao) {
            value = calc.sub(num1, num2);
        } else if (b.getId() == R.id.buttonMult) {
            value = calc.mult(num1, num2);
        } else if (b.getId() == R.id.buttonDiv) {
            value = calc.div(num1, num2);
        } else if (b.getId() == R.id.buttonSeno) {
            value = calc.seno(num1);
        } else if (b.getId() == R.id.buttonCos) {
            value = calc.cosseno(num1);
        } else if (b.getId() == R.id.buttonCos) {
            value = calc.tangente(num1);
        } else if (b.getId() == R.id.buttonModule) {
            value = calc.module(num1, num2);
        }else if (b.getId() == R.id.buttonSetOperador1) {
            setNum1(50);
        }else if (b.getId() == R.id.buttonGetOperador1) {
            getNum1();
        }
        number3EditText.setText(Double.toString(value));
    }

    private void setNum1(double newNum1){
        TextView number1EditText = findViewById(R.id.editText);
        String newNumber = Double.toString(newNum1);
        number1EditText.setText(newNumber);
        System.out.println("Yay apliquei o numero: " + newNum1 + " ao primeiro operador");
    }

    private double getDouble(String num) {
        return Double.parseDouble(num);
    }

    private double getNum1() {
        TextView number1EditText = findViewById(R.id.editText);
        String sNum1 = number1EditText.getText().toString();
        double num1 = getDouble(sNum1);
        System.out.println("Yay fui buscar o valor" + sNum1 + " do primeiro operador!");
        return num1;
    }
    @Override
    public void onWindowFocusChanged(boolean hasFocus) {
        super.onWindowFocusChanged(hasFocus);
        animDraw.start();
        animDraw2.start();
    }


    private void setContext(int id) {
        TextView title;
        int fc = 0, bc = 0, textColor = 0;
        View container = (View) findViewById(R.id.container);
        if (id == R.id.imageButtonPorto) {
            fc = R.color.colorBlue;
            bc = R.color.colorBlue;
            textColor = R.color.colorWhite;

            container.setBackgroundResource(R.drawable.estadioporto);
            title = (TextView) findViewById(R.id.title);
            title.setTextColor(ContextCompat.getColor(getBaseContext(), fc));

            title = (TextView) findViewById(R.id.textViewOperand1);
            title.setTextColor(ContextCompat.getColor(getBaseContext(), fc));

            title = (TextView) findViewById(R.id.textViewOperand3);
            title.setTextColor(ContextCompat.getColor(getBaseContext(), fc));

            title = (TextView) findViewById(R.id.textResult);
            title.setTextColor(ContextCompat.getColor(getBaseContext(), fc));

            title = (TextView) findViewById(R.id.editText);
            title.setTextColor(ContextCompat.getColor(getBaseContext(), textColor));
            title.setBackgroundColor(ContextCompat.getColor(getBaseContext(), bc));

            title = (TextView) findViewById(R.id.editText2);
            title.setTextColor(ContextCompat.getColor(getBaseContext(), textColor));
            title.setBackgroundColor(ContextCompat.getColor(getBaseContext(), bc));

            title = (TextView) findViewById(R.id.buttonSoma);
            title.setTextColor(ContextCompat.getColor(getBaseContext(), fc));

            title = (TextView) findViewById(R.id.buttonSubtracao);
            title.setTextColor(ContextCompat.getColor(getBaseContext(), fc));

            title = (TextView) findViewById(R.id.buttonMult);
            title.setTextColor(ContextCompat.getColor(getBaseContext(), fc));

            title = (TextView) findViewById(R.id.buttonDiv);
            title.setTextColor(ContextCompat.getColor(getBaseContext(), fc));

            title = (TextView) findViewById(R.id.buttonCos);
            title.setTextColor(ContextCompat.getColor(getBaseContext(), fc));

            title = (TextView) findViewById(R.id.buttonSeno);
            title.setTextColor(ContextCompat.getColor(getBaseContext(), fc));

            title = (TextView) findViewById(R.id.buttonTangente);
            title.setTextColor(ContextCompat.getColor(getBaseContext(), fc));

            title = (TextView) findViewById(R.id.buttonModule);
            title.setTextColor(ContextCompat.getColor(getBaseContext(), fc));

            title = (TextView) findViewById(R.id.buttonGetOperador1);
            title.setTextColor(ContextCompat.getColor(getBaseContext(), fc));

            title = (TextView) findViewById(R.id.buttonSetOperador1);
            title.setTextColor(ContextCompat.getColor(getBaseContext(), fc));

        } else if (id == R.id.imageButtonSLB) {
            fc = R.color.colorRed;
            bc = R.color.colorRed;
            textColor = R.color.colorWhite;

            container.setBackgroundResource(R.drawable.estadioluz);
            title = findViewById(R.id.title);
            title.setTextColor(ContextCompat.getColor(getBaseContext(), fc));

            title = (TextView) findViewById(R.id.textViewOperand1);
            title.setTextColor(ContextCompat.getColor(getBaseContext(), fc));

            title = (TextView) findViewById(R.id.editText);
            title.setTextColor(ContextCompat.getColor(getBaseContext(), textColor));
            title.setBackgroundColor(ContextCompat.getColor(getBaseContext(), bc));

            title = (TextView) findViewById(R.id.textViewOperand3);
            title.setTextColor(ContextCompat.getColor(getBaseContext(), fc));

            title = (TextView) findViewById(R.id.editText2);
            title.setTextColor(ContextCompat.getColor(getBaseContext(), textColor));
            title.setBackgroundColor(ContextCompat.getColor(getBaseContext(), bc));

            title = (TextView) findViewById(R.id.textResult);
            title.setTextColor(ContextCompat.getColor(getBaseContext(), fc));

            title = (TextView) findViewById(R.id.buttonSoma);
            title.setTextColor(ContextCompat.getColor(getBaseContext(), fc));

            title = (TextView) findViewById(R.id.buttonSubtracao);
            title.setTextColor(ContextCompat.getColor(getBaseContext(), fc));

            title = (TextView) findViewById(R.id.buttonMult);
            title.setTextColor(ContextCompat.getColor(getBaseContext(), fc));

            title = (TextView) findViewById(R.id.buttonDiv);
            title.setTextColor(ContextCompat.getColor(getBaseContext(), fc));

            title = (TextView) findViewById(R.id.buttonCos);
            title.setTextColor(ContextCompat.getColor(getBaseContext(), fc));

            title = (TextView) findViewById(R.id.buttonSeno);
            title.setTextColor(ContextCompat.getColor(getBaseContext(), fc));

            title = (TextView) findViewById(R.id.buttonTangente);
            title.setTextColor(ContextCompat.getColor(getBaseContext(), fc));

            title = (TextView) findViewById(R.id.buttonModule);
            title.setTextColor(ContextCompat.getColor(getBaseContext(), fc));

            title = (TextView) findViewById(R.id.buttonGetOperador1);
            title.setTextColor(ContextCompat.getColor(getBaseContext(), fc));

            title = (TextView) findViewById(R.id.buttonSetOperador1);
            title.setTextColor(ContextCompat.getColor(getBaseContext(), fc));

        } else if (id == R.id.imageButtonSCP) {
            fc = R.color.colorGreen;
            bc = R.color.colorGreen;
            textColor = R.color.colorWhite;

            container.setBackgroundResource(R.drawable.estadioscp);
            title = (TextView) findViewById(R.id.title);
            title.setTextColor(ContextCompat.getColor(getBaseContext(), fc));

            title = (TextView) findViewById(R.id.editText);
            title.setTextColor(ContextCompat.getColor(getBaseContext(), textColor));
            title.setBackgroundColor(ContextCompat.getColor(getBaseContext(), bc));

            title = (TextView) findViewById(R.id.textViewOperand1);
            title.setTextColor(ContextCompat.getColor(getBaseContext(), fc));

            title = (TextView) findViewById(R.id.editText2);
            title.setTextColor(ContextCompat.getColor(getBaseContext(), textColor));
            title.setBackgroundColor(ContextCompat.getColor(getBaseContext(), bc));

            title = (TextView) findViewById(R.id.textViewOperand3);
            title.setTextColor(ContextCompat.getColor(getBaseContext(), fc));

            title = (TextView) findViewById(R.id.textResult);
            title.setTextColor(ContextCompat.getColor(getBaseContext(), fc));

            title = (TextView) findViewById(R.id.buttonSoma);
            title.setTextColor(ContextCompat.getColor(getBaseContext(), fc));

            title = (TextView) findViewById(R.id.buttonSubtracao);
            title.setTextColor(ContextCompat.getColor(getBaseContext(), fc));

            title = (TextView) findViewById(R.id.buttonMult);
            title.setTextColor(ContextCompat.getColor(getBaseContext(), fc));

            title = (TextView) findViewById(R.id.buttonDiv);
            title.setTextColor(ContextCompat.getColor(getBaseContext(), fc));

            title = (TextView) findViewById(R.id.buttonCos);
            title.setTextColor(ContextCompat.getColor(getBaseContext(), fc));

            title = (TextView) findViewById(R.id.buttonSeno);
            title.setTextColor(ContextCompat.getColor(getBaseContext(), fc));

            title = (TextView) findViewById(R.id.buttonTangente);
            title.setTextColor(ContextCompat.getColor(getBaseContext(), fc));

            title = (TextView) findViewById(R.id.buttonModule);
            title.setTextColor(ContextCompat.getColor(getBaseContext(), fc));

            title = (TextView) findViewById(R.id.buttonGetOperador1);
            title.setTextColor(ContextCompat.getColor(getBaseContext(), fc));

            title = (TextView) findViewById(R.id.buttonSetOperador1);
            title.setTextColor(ContextCompat.getColor(getBaseContext(), fc));
        }
    }
}

File: Projects/ThematicCalculator/app/src/main/java/dam_a45125/thematiccalculator/MainActivity.kt
package dam_a45125.thematiccalculator

import android.annotation.SuppressLint
import android.graphics.drawable.AnimationDrawable
import android.os.Bundle
import android.view.View
import android.widget.Button
import android.widget.ImageButton
import android.widget.ImageView
import android.widget.TextView
import androidx.appcompat.app.AppCompatActivity
import androidx.core.content.ContextCompat

class MainActivity : AppCompatActivity(), View.OnClickListener {
    private var slbIcon: ImageButton? = null
    private var scpIcon: ImageButton? = null
    private var portoIcon: ImageButton? = null
    private var bSoma: Button? = null
    private var bSub: Button? = null
    private var bMult: Button? = null
    private var bDiv: Button? = null
    private var bCosseno: Button? = null
    private var bSeno: Button? = null
    private var bMod: Button? = null
    private var bTangente: Button? = null
    private var setButton: Button? = null
    private var getButton: Button? = null
    private var calc: Calculator? = null
    private var animDraw: AnimationDrawable? = null
    private var animDraw2: AnimationDrawable? = null

    @SuppressLint("ClickableViewAccessibility")
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        slbIcon = findViewById<View>(R.id.imageButtonSLB) as ImageButton
        scpIcon = findViewById<View>(R.id.imageButtonSCP) as ImageButton
        portoIcon = findViewById<View>(R.id.imageButtonPorto) as ImageButton
        bSoma = findViewById<View>(R.id.buttonSoma) as Button
        bSub = findViewById<View>(R.id.buttonSubtracao) as Button
        bMult = findViewById<View>(R.id.buttonMult) as Button
        bDiv = findViewById<View>(R.id.buttonDiv) as Button
        bCosseno = findViewById<View>(R.id.buttonCos) as Button
        bSeno = findViewById<View>(R.id.buttonSeno) as Button
        bTangente = findViewById<View>(R.id.buttonTangente) as Button
        bMod = findViewById<View>(R.id.buttonModule) as Button
        setButton = findViewById<View>(R.id.buttonGetOperador1) as Button
        getButton = findViewById<View>(R.id.buttonSetOperador1) as Button
        calc = Calculator()
        val imageView = findViewById<View>(R.id.imageAnim) as ImageView
        imageView.setBackgroundResource(R.drawable.animation)
        animDraw = imageView.background as AnimationDrawable
        val imageView2 = findViewById<View>(R.id.imageAnim2) as ImageView
        imageView2.setBackgroundResource(R.drawable.animations_network)
        animDraw2 = imageView2.background as AnimationDrawable
        slbIcon!!.setOnTouchListener { arg0, arg1 ->
            setContext(R.id.imageButtonSLB)
            false
        }
        scpIcon!!.setOnTouchListener { v, event ->
            setContext(R.id.imageButtonSCP)
            false
        }
        portoIcon!!.setOnTouchListener { v, event ->
            setContext(R.id.imageButtonPorto)
            false
        }
        bSoma!!.setOnClickListener(this)
        bSub!!.setOnClickListener(this)
        bMult!!.setOnClickListener(this)
        bDiv!!.setOnClickListener(this)
        bCosseno!!.setOnClickListener(this)
        bSeno!!.setOnClickListener(this)
        bTangente!!.setOnClickListener(this)
        bMod!!.setOnClickListener(this)
        setButton!!.setOnClickListener(this)
        getButton!!.setOnClickListener(this)
    }

    override fun onClick(v: View) {
        var calc = Calculator()
        var number1EditText = findViewById<TextView>(R.id.editText)
        var number2EditText = findViewById<TextView>(R.id.editText2)
        var number3EditText = findViewById<TextView>(R.id.textResult)
        var sNum1 = number1EditText.text.toString()
        var sNum2 = number2EditText.text.toString()
        var num1 = getDouble(sNum1)
        var num2 = getDouble(sNum2)
        var b = v as Button
        var value = 0.0

        if (b.id == R.id.buttonSoma) {
            value = calc.sum(num1, num2)
        } else if (b.id == R.id.buttonSubtracao) {
            value = calc.sub(num1, num2)
        } else if (b.id == R.id.buttonMult) {
            value = calc.mult(num1, num2)
        } else if (b.id == R.id.buttonDiv) {
            value = calc.div(num1, num2)
        } else if (b.id == R.id.buttonSeno) {
            value = calc.seno(num1)
        } else if (b.id == R.id.buttonCos) {
            value = calc.cosseno(num1)
        } else if (b.id == R.id.buttonCos) {
            value = calc.tangente(num1)
        } else if (b.id == R.id.buttonModule) {
            value = calc.module(num1, num2)
        } else if (b.id == R.id.buttonSetOperador1) {
            value = num1.toDouble()
            println("Yay apliquei o $num1 ao resultado final")
        } else if (b.id == R.id.buttonGetOperador1) {
            getDouble(num1.toString())
            println("Yay fui buscar o valor $num1 ao primeiro operador" )
        }
        number3EditText.text = value.toString()
    }

    private fun getDouble(num: String): Double {
        return num.toDouble()
    }

    override fun onWindowFocusChanged(hasFocus: Boolean) {
        super.onWindowFocusChanged(hasFocus)
        animDraw!!.start()
        animDraw2!!.start()
    }

    private fun setContext(id: Int) {
        var title: TextView
        var fc = 0
        var bc = 0
        var textColor = 0
        val container = findViewById<View>(R.id.container)
        if (id == R.id.imageButtonPorto) {
            fc = R.color.colorBlue
            bc = R.color.colorBlue
            textColor = R.color.colorWhite
            container.setBackgroundResource(R.drawable.estadioporto)
            title = findViewById<View>(R.id.title) as TextView
            title.setTextColor(ContextCompat.getColor(baseContext, fc))
            title = findViewById<View>(R.id.textViewOperand1) as TextView
            title.setTextColor(ContextCompat.getColor(baseContext, fc))
            title = findViewById<View>(R.id.textViewOperand3) as TextView
            title.setTextColor(ContextCompat.getColor(baseContext, fc))
            title = findViewById<View>(R.id.textResult) as TextView
            title.setTextColor(ContextCompat.getColor(baseContext, fc))
            title = findViewById<View>(R.id.editText) as TextView
            title.setTextColor(ContextCompat.getColor(baseContext, textColor))
            title.setBackgroundColor(ContextCompat.getColor(baseContext, bc))
            title = findViewById<View>(R.id.editText2) as TextView
            title.setTextColor(ContextCompat.getColor(baseContext, textColor))
            title.setBackgroundColor(ContextCompat.getColor(baseContext, bc))
            title = findViewById<View>(R.id.buttonSoma) as TextView
            title.setTextColor(ContextCompat.getColor(baseContext, fc))
            title = findViewById<View>(R.id.buttonSubtracao) as TextView
            title.setTextColor(ContextCompat.getColor(baseContext, fc))
            title = findViewById<View>(R.id.buttonMult) as TextView
            title.setTextColor(ContextCompat.getColor(baseContext, fc))
            title = findViewById<View>(R.id.buttonDiv) as TextView
            title.setTextColor(ContextCompat.getColor(baseContext, fc))
            title = findViewById<View>(R.id.buttonCos) as TextView
            title.setTextColor(ContextCompat.getColor(baseContext, fc))
            title = findViewById<View>(R.id.buttonSeno) as TextView
            title.setTextColor(ContextCompat.getColor(baseContext, fc))
            title = findViewById<View>(R.id.buttonTangente) as TextView
            title.setTextColor(ContextCompat.getColor(baseContext, fc))
            title = findViewById<View>(R.id.buttonModule) as TextView
            title.setTextColor(ContextCompat.getColor(baseContext, fc))
            title = findViewById<View>(R.id.buttonGetOperador1) as TextView
            title.setTextColor(ContextCompat.getColor(baseContext, fc))
            title = findViewById<View>(R.id.buttonSetOperador1) as TextView
            title.setTextColor(ContextCompat.getColor(baseContext, fc))
        } else if (id == R.id.imageButtonSLB) {
            fc = R.color.colorRed
            bc = R.color.colorRed
            textColor = R.color.colorWhite
            container.setBackgroundResource(R.drawable.estadioluz)
            title = findViewById(R.id.title)
            title.setTextColor(ContextCompat.getColor(baseContext, fc))
            title = findViewById<View>(R.id.textViewOperand1) as TextView
            title.setTextColor(ContextCompat.getColor(baseContext, fc))
            title = findViewById<View>(R.id.editText) as TextView
            title.setTextColor(ContextCompat.getColor(baseContext, textColor))
            title.setBackgroundColor(ContextCompat.getColor(baseContext, bc))
            title = findViewById<View>(R.id.textViewOperand3) as TextView
            title.setTextColor(ContextCompat.getColor(baseContext, fc))
            title = findViewById<View>(R.id.editText2) as TextView
            title.setTextColor(ContextCompat.getColor(baseContext, textColor))
            title.setBackgroundColor(ContextCompat.getColor(baseContext, bc))
            title = findViewById<View>(R.id.textResult) as TextView
            title.setTextColor(ContextCompat.getColor(baseContext, fc))
            title = findViewById<View>(R.id.buttonSoma) as TextView
            title.setTextColor(ContextCompat.getColor(baseContext, fc))
            title = findViewById<View>(R.id.buttonSubtracao) as TextView
            title.setTextColor(ContextCompat.getColor(baseContext, fc))
            title = findViewById<View>(R.id.buttonMult) as TextView
            title.setTextColor(ContextCompat.getColor(baseContext, fc))
            title = findViewById<View>(R.id.buttonDiv) as TextView
            title.setTextColor(ContextCompat.getColor(baseContext, fc))
            title = findViewById<View>(R.id.buttonCos) as TextView
            title.setTextColor(ContextCompat.getColor(baseContext, fc))
            title = findViewById<View>(R.id.buttonSeno) as TextView
            title.setTextColor(ContextCompat.getColor(baseContext, fc))
            title = findViewById<View>(R.id.buttonTangente) as TextView
            title.setTextColor(ContextCompat.getColor(baseContext, fc))
            title = findViewById<View>(R.id.buttonModule) as TextView
            title.setTextColor(ContextCompat.getColor(baseContext, fc))
            title = findViewById<View>(R.id.buttonGetOperador1) as TextView
            title.setTextColor(ContextCompat.getColor(baseContext, fc))
            title = findViewById<View>(R.id.buttonSetOperador1) as TextView
            title.setTextColor(ContextCompat.getColor(baseContext, fc))
        } else if (id == R.id.imageButtonSCP) {
            fc = R.color.colorGreen
            bc = R.color.colorGreen
            textColor = R.color.colorWhite
            container.setBackgroundResource(R.drawable.estadioscp)
            title = findViewById<View>(R.id.title) as TextView
            title.setTextColor(ContextCompat.getColor(baseContext, fc))
            title = findViewById<View>(R.id.editText) as TextView
            title.setTextColor(ContextCompat.getColor(baseContext, textColor))
            title.setBackgroundColor(ContextCompat.getColor(baseContext, bc))
            title = findViewById<View>(R.id.textViewOperand1) as TextView
            title.setTextColor(ContextCompat.getColor(baseContext, fc))
            title = findViewById<View>(R.id.editText2) as TextView
            title.setTextColor(ContextCompat.getColor(baseContext, textColor))
            title.setBackgroundColor(ContextCompat.getColor(baseContext, bc))
            title = findViewById<View>(R.id.textViewOperand3) as TextView
            title.setTextColor(ContextCompat.getColor(baseContext, fc))
            title = findViewById<View>(R.id.textResult) as TextView
            title.setTextColor(ContextCompat.getColor(baseContext, fc))
            title = findViewById<View>(R.id.buttonSoma) as TextView
            title.setTextColor(ContextCompat.getColor(baseContext, fc))
            title = findViewById<View>(R.id.buttonSubtracao) as TextView
            title.setTextColor(ContextCompat.getColor(baseContext, fc))
            title = findViewById<View>(R.id.buttonMult) as TextView
            title.setTextColor(ContextCompat.getColor(baseContext, fc))
            title = findViewById<View>(R.id.buttonDiv) as TextView
            title.setTextColor(ContextCompat.getColor(baseContext, fc))
            title = findViewById<View>(R.id.buttonCos) as TextView
            title.setTextColor(ContextCompat.getColor(baseContext, fc))
            title = findViewById<View>(R.id.buttonSeno) as TextView
            title.setTextColor(ContextCompat.getColor(baseContext, fc))
            title = findViewById<View>(R.id.buttonTangente) as TextView
            title.setTextColor(ContextCompat.getColor(baseContext, fc))
            title = findViewById<View>(R.id.buttonModule) as TextView
            title.setTextColor(ContextCompat.getColor(baseContext, fc))
            title = findViewById<View>(R.id.buttonGetOperador1) as TextView
            title.setTextColor(ContextCompat.getColor(baseContext, fc))
            title = findViewById<View>(R.id.buttonSetOperador1) as TextView
            title.setTextColor(ContextCompat.getColor(baseContext, fc))
        }
    }
}